src/components/ThemeProvider/README.md

```diff
@@ -141,6 +141,74 @@ A theme provider can be nested within the theme provider rendered by the app pro
 </AppProvider>
 ```
 
+### Theme provider with a color scheme and custom config nested within an app provider
+
+You can pass in a custom config to the theme provider to override default HSL values. Find the lightness value you need using www.hsluv.org
+
+```jsx
+<AppProvider i18n={{}}>
+  <TextContainer>
+    <Card
+      title="Shipment 1234"
+      secondaryFooterActions={[{content: 'Edit shipment'}]}
+      primaryFooterAction={{content: 'Add tracking number'}}
+    >
+      <Card.Section title="Items">
+        <List>
+          <List.Item>1 × Oasis Glass, 4-Pack</List.Item>
+          <List.Item>1 × Anubis Cup, 2-Pack</List.Item>
+        </List>
+      </Card.Section>
+    </Card>
+    <ThemeProvider
+      theme={{
+        colors: {
+          surface: '#054948',
+        },
+        config: {
+          surface: [
+            {
+              name: 'surface',
+              description:
+                'For use as a background color, in components such as Card, Modal, and Popover.',
+              light: {lightness: 27.5},
+              dark: {lightness: 27.5},
+              meta: {
+                figmaName: 'Surface/Default',
+              },
+            },
+          ],
+          onSurface: [
+            {
+              name: 'text',
+              description: 'For use as a text color.',
+              light: {lightness: 100},
+              dark: {lightness: 100},
+              meta: {
+                figmaName: 'Text/Default',
+              },
+            },
+          ],
```

Comment:
Where can one read more about these config values and understand what each means?

Reply:
Should we add some documentation around what `lightness` means, and what `figmaName` means, etc? Or are those considered domain knowledge?

Reply:
I feel like between the color docs file and figma it's easy enough to find what you're looking for. 

Lightness, saturation, and hue on the other hand may be intuitive to some more than others. I'm not sure where exactly the documentation for that would live off the top of my head though.

Feel free to create a PR in a place you see fit and ping me as a reviewer